pkg/arguments/arguments_test.go

```diff
@@ -0,0 +1,48 @@
+package arguments
+
+import (
+	"github.com/spf13/cobra"
+	errors "github.com/zgalor/weberr"
+)
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Client", func() {
+	var (
+		cmd *cobra.Command
+	)
+
+	Context("Test PreprocessUnknownFlagsWithId func", func() {
+		BeforeEach(func() {
+			cmd = &cobra.Command{
+				Use:     "addon ID",
+				Aliases: []string{"addons", "add-on", "add-ons"},
+				Short:   "Edit add-on installation parameters on cluster",
+				Long:    "Edit the parameters on installed Red Hat managed add-ons on a cluster",
+				Example: `  # Edit the parameters of the Red Hat OpenShift logging operator add-on installation
+  rosa edit addon --cluster=mycluster cluster-logging-operator`,
+				DisableFlagParsing: true,
```

Comment:
NP : Can you add some dummy uses and stuff to this cmd, just want to ensure it is not confused in the future as a test for addon command. 

Also might be worth breaking into a helper function to allow us to easily call this for the other funcs in the arguments file that are missing tests :smile: